torch/csrc/jit/runtime/interpreter.cpp

```diff
@@ -1153,6 +1158,10 @@ struct InterpreterStateImpl : c10::intrusive_ptr_target {
     }
 
     ActiveFrame af(frames.back());
+    TLSCurrentInterpreterGuard g(this);
+    if (af.pc == 0) {
```

Comment:
`af.pc == 0` is a fragile check for the start of a function.  The bytecode generator might produce bytecode that jumps back to `pc == 0` so there are some rare scenarios where its 0 but the start of the function.

Reply:
The code right above this does this using `stack_start_` which is more stable.